skimage/transform/integral.py

```diff
@@ -21,15 +21,28 @@ def integral_image(image):
     S : ndarray
         Integral image/summed area table of same shape as input image.
 
+    Notes
+    -----
+    For better accuracy and to avoid potential overflow, the datatype of the
+    output may differ from the input when the default dtype of None is used.
```

Comment:
```suggestion
    output may differ from the input's when the default dtype of None is used.
```
or "that of the input"